section2/reflection.md

```diff
@@ -2,14 +2,34 @@
+
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
 
+  * It felt fine, I was able to return to earlier resources with more explicit instructions the first time I needed to run `node directory/filename.js` and it felt satisfying to implement it in subsequent sections without need to review. I like it because it takes a way the training wheels, it ensures that I have it memorized.  
+
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
 
-5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
+  * A conditional is code that uses `if,` `else if` and `else` to evaluate a (or multiple) condition(s). If one of the conditions is met, the code associated with it is executed and none of the subsequent conditions are executed.   
+
+  * **Example:** If the forecast calls for rain, bring an umbrella. If not rain but snow, bring a coat. In all other cases, bring sunglasses.
+
+  * A conditional is likely used by the YouTube algorithm to determine when to show videos. If a channel you subscribe to has uploaded a video today, display it on the front page. If you have already watched that video, do not display it.     
+
+5. How do you add multiple conditions to an `if` statement? In your own words, explain how the program reads them and determines what to do.
+
+  * To add multiple conditions to an `if` statement, add as many `else if` statements as needed. If the initial `if` returns a Boolean value of true (or false, if that is what is specified) then the code executes; if not, the `else if` is consulted and if its conditions are met, it executes; this continues until a condition is met or the final `else` is reached, at which point it executes whatever is subsequent to the else.
 
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+  * The `&&` operator is used to check that two conditions are met; the `||` operator is used to check that 1 of 2 conditions are met.
+
+7. What questions do you still have about `if` statements and/or functions?
+
+  * How do operators like `&&` and `||` interact with concatenation and interpolation? Are they ever necessary or is double parenthesis adequate to keep different kinds of variables from mixing poorly?
```

Comment:
I don't think that using `&&` or `||` inside of interpolation is super common. You could have a conditional statement and then use a variable inside of a string based on that condition, but you would usually keep the logic out of the string.